packages/nextjs/scaffold.config.ts

```diff
@@ -19,6 +24,21 @@ const scaffoldConfig = {
   // It's recommended to store it in an env variable:
   // .env.local for local testing, and in the Vercel/system env config for live apps.
   alchemyApiKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY || "oKxs-03sij-U_N0iOlrSsZFr29-IqbuF",
+
+  // Burner Wallet configuration
+  burnerWallet: {
+    // Set it to false to completely remove burner wallet from all networks
+    enabled: true,
+    // Only show the Burner Wallet when running on hardhat network
+    onlyLocal: false,
+  },
+
+  /**
+   * Auto connect:
+   * 1. If the user was connected into a wallet before, on page reload reconnect automatically
+   * 2. If user is not connected to any wallet:  On reload, connect to burner wallet if burnerWallet.enabled is true && burnerWallet.onlyLocal is false
+   */
+  walletAutoConnect: true,
```

Comment:
Good catch!!! This is not only burner wallet, so it should live here, not inside the burner wallet config ;)